tests/bluetooth/controller/ctrl_cte_req/src/main.c

```diff
@@ -616,6 +616,751 @@ void test_cte_req_reject_inv_ll_param_peripheral_remote(void)
 				  "Free CTX buffers %d", ctx_buffers_free());
 }
 
+/* Tests related with PHY update procedure and CTE request procedure "collision" */
+
+#define PREFER_S2_CODING 0U
+#define HOST_INITIATED 1U
+#define PHY_UPDATE_INSTANT_DELTA 6
+
+#define CHECK_PREF_PHY_STATE(_conn, _tx, _rx)                                                      \
+	do {                                                                                       \
+		zassert_equal(_conn.phy_pref_rx, _rx,                                              \
+			      "Preferred RX PHY mismatch %d (actual) != %d (expected)",            \
+			      _conn.phy_pref_rx, _rx);                                             \
+		zassert_equal(_conn.phy_pref_tx, _tx,                                              \
+			      "Preferred TX PHY mismatch %d (actual) != %d (expected)",            \
+			      _conn.phy_pref_tx, _tx);                                             \
+	} while (0)
+
+#define CHECK_CURRENT_PHY_STATE(_conn, _tx, _flags, _rx)                                           \
+	do {                                                                                       \
+		zassert_equal(_conn.lll.phy_rx, _rx,                                               \
+			      "Current RX PHY mismatch %d (actual) != %d (expected)",              \
+			      _conn.lll.phy_rx, _rx);                                              \
+		zassert_equal(_conn.lll.phy_tx, _tx,                                               \
+			      "Current TX PHY mismatch %d (actual) != %d (expected)",              \
+			      _conn.lll.phy_tx, _tx);                                              \
+		zassert_equal(_conn.lll.phy_rx, _rx,                                               \
+			      "Current Flags mismatch %d (actual) != %d (expected)",               \
+			      _conn.lll.phy_flags, _flags);                                        \
+	} while (0)
```

Comment:
Do these need to macros instead of functions?
Refer to https://www.kernel.org/doc/html/v4.10/process/coding-style.html#macros-enums-and-rtl

> Generally, inline functions are preferable to macros resembling functions.

And 
> Things to avoid when using macros:
    1. macros that affect control flow:

Reply:
Changed to functions.